testsuite/classlibrary/TestNodeProxy.sc

```diff
@@ -45,97 +44,124 @@ TestNodeProxy : UnitTest {
 		Server.default.sync;
 
 		this.assert(
-			Ndef(\x).loaded == false,
+			proxy.loaded == false,
 			"after server has started, NodeProxy shouldn't change by itself."
 		);
 
-		Ndef(\x).send;
+		proxy.send;
 
 		this.assert(x == nil,
 			"by calling 'send', SynthDef shouldn't be rebuilt");
 
-		this.assert(Ndef(\x).isPlaying,
+		this.assert(proxy.isPlaying,
 			"after send: NodeProxy group should be playing");
 
-		this.assert(Ndef(\x).loaded == true,
+		this.assert(proxy.loaded == true,
 			"after send: NodeProxy should have sent resources to the server (loaded)");
 
-		Ndef(\x).rebuild;
+		proxy.rebuild;
 
 
 		this.assert(x == "rebuilt",
 			"after rebuilt: synth function should have been called");
 
-		this.assert(Ndef(\x).loaded == true,
+		this.assert(proxy.loaded == true,
 			"after rebuild and server running: resources should be on the server");
 
 		// QUIT
 		s.quit;
 		x = nil;
-		Ndef(\x).rebuild;
+		proxy.rebuild;
 
 		this.assert(x == "rebuilt",
 			"after server quit and rebuild: synth function should have been called");
 
-		this.assert(Ndef(\x).loaded == false,
+		this.assert(proxy.loaded == false,
 			"after server quit: no resources should be on the server");
 
 		/* will be fixed in master
-		Ndef(\x).send;
-		this.assert(Ndef(\x).loaded != true,
+		proxy.send;
+		this.assert(proxy.loaded != true,
 			"after server quit: send should not assume server resources loaded");
 		*/
 
 		// CLEAR
-		Ndef.clear;
-		try { Ndef(\x).ar(s.options.numAudioBusChannels * 2) };
+		proxy.clear;
+		try { proxy.ar(s.options.numAudioBusChannels * 2) };
 
-		this.assert(Ndef(\x).isNeutral,
+		this.assert(proxy.isNeutral,
 			"trying to allocate more bus channels than available should leave the node proxy untouched"
 		);
 
 		// ELASTIC
 
-		Ndef(\x).reshaping = \elastic;
+		proxy.reshaping = \elastic;
 
 		try { Ndef(\x, { DC.ar(0 ! (s.options.numAudioBusChannels * 2)) }) };
 
-		this.assert(Ndef(\x).isNeutral,
+		this.assert(proxy.isNeutral,
 			"when elastic: trying to allocate more bus channels than available should leave the node proxy untouched"
 		);
 
-		Ndef(\x).ar(8);
+		proxy.ar(8);
 
-		this.assert(Ndef(\x).numChannels == 8,
+		this.assert(proxy.numChannels == 8,
 			"setting number of channels should change them"
 		);
 
 		try { Ndef(\x, { DC.ar(0 ! (s.options.numAudioBusChannels * 2)) }) };
 
-		this.assert(Ndef(\x).numChannels == 8,
+		this.assert(proxy.numChannels == 8,
 			"when elasic and initialised: trying to allocate more bus channels than available should leave the node proxy untouched"
 		);
 
 
+		proxy.clear;
+		s.quit;
+		s.remove;
+
 	}
 
 	test_fadeTime {
 
-		var s = Server.default;
-		s.quit;
+		var s = Server(this.class.name);
+		var proxy;
+
 		this.bootServer(s);
 		s.sync;
-		1.wait;
 
-		Ndef.clear;
-		Ndef(\x).fadeTime = 2;
-		Ndef(\x).reshaping = \elastic;
-		Ndef(\x).clear(1);
+		proxy = NodeProxy(s);
+		proxy.fadeTime = 2;
+		proxy.reshaping = \elastic;
+		proxy.clear(1);
 		1.01.wait;
```

Comment:
is there not a signal we can wait on here?

Reply:
`\clear` seems to be called immediately after `proxy.clear` regardless of `fadeTime`. I think this is an issue for another PR.

Reply:
I agree, the changes in this PR to other tests have only been about simple / minimal improvements. those should be in another PR by all rights anyway.